documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -2463,4 +2465,59 @@ To use a custom SPI implementation, it must be located in a seperate .jar file t
 [TIP]
 Fore more details: There's the above example is present in our our examples repository (https://github.com/mapstruct/mapstruct-examples).
 
+[mapping-exclusion-provider]
+=== Mapping Exclusion Provider
 
+MapStruct offers the possibility to override the `MappingExclusionProvider` via the Service Provider Interface (SPI).
+A nice example is to not allow MapStruct to create an automatic sub-mapping for a certain type,
```

Comment:
In the [`Mapping object references`](http://mapstruct.org/documentation/dev/reference/html/#mapping-object-references) section we have the routine that MapStruct uses to create a mapping. The last step before raising an error is:

> * If no such method was found MapStruct will try to generate an internal method that will do the mapping between the source and target attributes

Will it suffice if we have:

* If no such method was found MapStruct will try to generate an automatic sub-mapping method that will do the mapping between the source and target attributes

instead? Or you think we need more context.  I can also write the routine steps in the `MappingExclusionProvider` as well, so when someone sees the interface they can have more content.